src/exceptions/x11vnc-openssl-exception.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="x11vnc-openssl-exception"
+   name="x11vnc OpenSSL Exception" listVersionAdded="3.19">
+      <crossRefs>
+         <crossRef>https://github.com/LibVNC/x11vnc/blob/master/src/8to24.c#L22</crossRef>
+      </crossRefs>
+     
+      <notes>This is very similar to openvpn-openssl-exception with a couple key differences.
+      </notes>
+      <text>
+         <p>
+            In addition, as a special exception, Karl J. Runge gives permission
+            to link the code of its release of x11vnc with the OpenSSL project's
+            "OpenSSL" library (or with modified versions of it that use the
+            same license as the "OpenSSL" library), and distribute the linked
+            executables. You must obey the GNU General Public License in all
+            respects for all of the code used other than "OpenSSL". If you
+            modify this file, you may extend this exception to your version
+            of the file, but you are not obligated to do so. If you do not
+            wish to do so, delete this exception statement from your version.
+         </p>
+      </text>
+   </license>
```

Comment:
This would also change from `</license>` to `</exception>`.

Reply:
done